.github/workflows/integration.yaml

```diff
@@ -27,6 +27,15 @@ jobs:
           - 9091:9091
           - 9092:9092
 
+      kafka_confluent:
```

Comment:
Should we rename the one above from "kafka" to "kafka_sarama" to be consistent with the rest of the naming?